src/SSVOpenHexagon/Core/HGUpdate.cpp

```diff
@@ -266,6 +262,23 @@ void HexagonGame::update(ssvu::FT mFT)
         fpsWatcher.update();
     }
 }
+
+void HexagonGame::updateWalls(ssvu::FT mFT){
+    const auto playerPos{player.getPosition()};
+
+    for(CWall& wall : walls) {
+        player.push(*this, wall);
+
+        if (wall.moveTowardsCenter(*this, centerPos, mFT)) {
```

Comment:
What I mean above is something like

```suggestion
        wall.moveTowardsCenter(*this, centerPos, mFT);
        if (wall.isOverlapping(player)) {
```

Reply:
alright, will change all of that soon.